docs/RELEASING.md

```diff
@@ -11,28 +11,50 @@
    k9mail.keyPassword=<password 'k9mail@tb-android' in 1Password>
    ```
 
-## One-time setup for F-Droid builds
+### One-time setup for F-Droid builds
 
-1. Install _fdroidserver_ by following the [installation instructions](https://f-droid.org/docs/Installing_the_Server_and_Repo_Tools).
-2. [Sign up for a Gitlab account](https://gitlab.com/users/sign_up) and fork the [fdroiddata](https://gitlab.com/fdroid/fdroiddata) repository.
+1. Install _fdroidserver_ by following
+   the [installation instructions](https://f-droid.org/docs/Installing_the_Server_and_Repo_Tools).
+   1. On MacOS, it's best to install the latest version from source, because the version in Homebrew has some issues.
+      1. Install the android command line tools if not available already.
+         ```
+         brew install --cask android-commandlinetools
+         ```
+      2. Install latest _fdroidserver_ from source:
+         ```
+         python -m venv fdroidserver-env
+         source fdroidserver-env/bin/activate
+         pip install git+https://gitlab.com/fdroid/fdroidserver.git
+         ```
+      3. To use _fdroidserver_ from the command line, you need to activate the virtual environment before each use:
+         ```
+         source fdroidserver-env/bin/activate
+         ```
+      4. To deactivate the virtual environment:
+         ```
+         deactivate
+         ```
+2. [Sign up for a Gitlab account](https://gitlab.com/users/sign_up) and fork
+   the [fdroiddata](https://gitlab.com/fdroid/fdroiddata) repository.
 3. Clone your fork of the _fdroiddata_ repository.
 
 ## Release a beta version
 
-1. Update versionCode and versionName in `app/k9mail/build.gradle`
+1. Update versionCode and versionName in `app-k9mail/build.gradle.kts`
 2. Create change log entries in
    - `app/ui/legacy/src/main/res/raw/changelog_master.xml`
-   - `fastlane/metadata/android/en-US/changelogs/${versionCode}.txt`
+   - `app-k9mail/fastlane/metadata/android/en-US/changelogs/${versionCode}.txt`
      Use past tense. Try to keep them high level. Focus on the user (experience).
 3. Commit the changes. Message: "Version $versionName"
-4. Run `gradlew clean :app:k9mail:assembleRelease --no-build-cache`
+4. Run `./gradlew clean :app-k9mail:assembleRelease --no-build-cache --no-configuration-cache`
 5. Update an existing installation to make sure the app is signed with the proper key and runs on a real device.
    ```
-   adb install -r app/k9mail/build/outputs/apk/release/k9mail-release.apk
+   adb install -r app-k9mail/build/outputs/apk/release/app-k9mail-release-unsigned.apk
```

Comment:
When the keystore is properly configured, this probably won't have the `-unsigned` suffix.